### PR TITLE
Document shared items feature for contacts

### PR DESCRIPTION
**Pull request by GretaD**
### ☑️ Resolves

* Fix https://github.com/nextcloud/contacts/pull/3670

### 🖼️ Screenshots
Screenshot from 2023-11-08 10-52-29

### CODE REVIEW

File: user_manual/groupware/contacts.rst [hunk 1]
```diff
@@ -182,3 +182,7 @@ Circle options
 ~~~~~~~~~~~~~~
 
 Various self-explanatory options are available to configure a circle, to manage invites and membership, visibility of the circle, allowance of other circle membership and password protection.
+
+Shared items
+~~~~~~~~~~~~
+The contact app supports shared item starting from version 5.5.0. Media, calendar events, talk chat room between you and another contact or shared deck cards, will be shown on the contact details. You need to have the app activated in order to see the shared items of that app. Right now, we only support the shared items between 2 contacts.
```
Comment (ChristophWurst) [inline user_manual/groupware/contacts.rst:188]: * The user can't enable an app. Drop the note about enabled app.
* Use https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded to render when the feature was added.
* Add note that this only works for contacts in the *system address book*

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Add a screenshot of the rendered page

File: user_manual/groupware/contacts.rst [hunk 2]
```diff
@@ -182,3 +182,9 @@ Circle options
 ~~~~~~~~~~~~~~
 
 Various self-explanatory options are available to configure a circle, to manage invites and membership, visibility of the circle, allowance of other circle membership and password protection.
+
+Shared items
+~~~~~~~~~~~~
+.. versionadded:: 5.5
+.. figure:: ../images/shared-items.png
```
Comment (ChristophWurst) [inline user_manual/groupware/contacts.rst:189]: I meant a screenshot for the PR description but this is nice too :)

**Review by ChristophWurst — COMMENTED** [review]
:sunglasses:

**Review by ChristophWurst — COMMENTED** [review]
:+1: Add a screenshot of the modified docs page to the PR then we can merge

**Comment by GretaD** [comment]
> 👍 Add a screenshot of the modified docs page to the PR then we can merge

the way how you add images on github has changed. I couldnt find how to do it and assumed is a temporary issue.

Screenshot from 2023-11-08 10-52-29

**Review by ChristophWurst — APPROVED** [review]
Very nice

**Comment by ChristophWurst** [comment]
This is only to be merged when the feature PR is in, in case there is a delay

**Review by hamza221 — APPROVED** [review]

**Pull request merged**